conftest.py

```diff
@@ -86,10 +86,8 @@ def setup_mender_configured(
         # If mender is already present, do nothing.
         return
 
-    url = (
-        "https://downloads.mender.io/repos/debian/pool/main/m/mender-client/mender-client_%s-1_armhf.deb"
-        % mender_deb_version
-    )
+    url = "https://downloads.mender.io/repos/debian/pool/main/m/mender-client/"
+    url += f"mender-client_{mender_deb_version}-1%2bdebian%2bbuster_armhf.deb"
```

Comment:
Question:
Looks good, but are we always going to hard-code the debian-revision?

Reply:
Right now we don't have the means to bump the Debian version. It is hardcoded (generated at runtime, really) in `mender-dist-packages`. This deserves a separate nice coffee discussion :)

Reply:
:coffee: